Musicca/app/src/main/java/com/example/musicca/activities/SongPlaylistActivity.java

```diff
@@ -48,14 +52,16 @@
     private ImageView ivNext;
     private Button btnReturnPlaylist;
 
-    private String albumUrl;
     private String playlistObjectId;
-    private String songObjectId;
 
     private static final String CLIENT_ID = "22793b7728c54470b8d117506f9574c5";
     private static final String REDIRECT_URI = "com.musicca://callback";
     private SpotifyAppRemote mSpotifyAppRemote;
 
+    // boolean to determine is song is being played for the first time, on resume or on pause
+    private boolean isPlay = false;
```

Comment:
The name is a little confusing. What does this do exactly? You might want to rename it to make it more clear.

Reply:
I have renamed the booleans to isSongPlaying and isSongPaused